### ISSUE TITLE
write Field II mesh interpolation function

### ISSUE THREAD

**Issue by mlp6** [mlp6/fem#9]
instead of having to always re-run Field II when using a new mesh, we should have a function that will take an existing 3D Field II intensity output and 3D map it onto a new grid

**Comment by mlp6** [comment]
http://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.interpolate.griddata.html

**Comment by mlp6** [comment]
This seems to be more important of a feature now for a few reasons:


1. Field II doesn't play nicely with RAM and meshes that have large absolute extents (regardless of mesh density), which reared its head when trying to simulate the 4V1c
2. `field2dyna` doesn't need to chug through tons of points when doing a shear wave simulation and the vast majority of the nodes are outside the footprint of the active array aperture
3. Migrating to the C-version of Field II Pro would be easier to just specify a 3D array of points in the source code than to have to read in the `nodes.dyn` file.

**Comment by mlp6** [comment]
@mjr52 your task to integrate the HFR-ARFI beams in falls under this Issue.  Would be nice to make something that works for both applications (HFR-ARFI and the use-case detailed above).

**Comment by mlp6** [comment]
@jrichardson97 your branch that I'm merging in (https://github.com/mlp6/fem/pull/121) does this now, correct?

**Comment by jrichardson97** [comment]
Yes. I'll make an example with a sample load file and the interpolation constraints

**Comment by mlp6** [comment]
@jrichardson97 you will need to create a new branch off of `main` for that as I just merged in your large pull request after some tweaks to the testing environment.  I'll let you close this Issue when that PR is ready to go.
